README.rst

```diff
@@ -1,207 +1,192 @@
+pygame
+======
 
+|TravisBuild| |AppVeyorBuild| |LaunchpadBuild|
+|PyPiVersion| |PyPiLicense| |Python2| |Python3| |GithubCommits|
+|LGTMAlerts| |LGTMGradePython| |LGTMGradeC| |Coverity|
 
-About pygame
-============
+This library is cross-platform and designed to make it easy to
```

Comment:
How about "The pygame library is a cross-platform"

In the last years we try to use "pygame" everywhere in the last couple of years to try and reinforce the point that lower case matters in python. We also have a better introduction that maybe can be used here... https://github.com/pygame/pygameweb/blob/master/pygameweb/templates/news/welcome.html It includes various links.

Reply:
I rephrased this as

> pygame is a Free and Open Source cross-platform library for the development of multimedia applications like video games on Python. It uses the Simple DirectMedia Layer library and several other popular libraries to abstract most common functions and makes writing these program a more intuitive task.

Does that sound better?